chapter20/README.md

```diff
@@ -20,7 +20,9 @@ let up_case = fun s -> Writer(String.uppercase_ascii s, "up_case ")
 (* Depends on OCaml library Base - #require "base" *)
 module Vlen(F : Functor with type 'a t = 'a list) = struct
   
-  let summable = (module Float:Base.Container_intf.Summable with type t = float)
+  open Base
+  
+  let summable = (module Float:Base__.Container_intf.Summable with type t = float)
```

Comment:
I had to write `Base__` instead of `Base.Container` to get the code to compile. How do I make Container visible via Base?